test/langtools/jdk/javadoc/doclet/testSingletonLists/TestSingletonLists.java

```diff
@@ -290,10 +283,15 @@ public void endElement(String name) {
             switch (name) {
                 case "ul": case "ol": {
                     Map<String,Integer> c = counts.pop();
-                    if (c.get("li") == 0) {
+                    if (inSeeList) {
+                        // Ignore "Constant Field Values" @see items for final fields created by javadoc
```

Comment:
I think the comment is too specific. Won't the problem also be triggered for classes with a single `@see` and n o constant values?

Reply:
Yes, the problem could be triggered by various circumstances, but in this particular test (which doesn't contain any explicit `@see` tags) it is just the automatically generated "see also" link to the constant field values page. The fields in question can't be made non-final because they are in annotation interfaces, btw.